src/website/typography/Heading/Heading.js

```diff
@@ -0,0 +1,54 @@
+export default function Heading({
+    typeStyle = "heading-lg",
+    level = 'h2',
+    textColor = 'text-primary-800',
```

Comment:
Maybe `textColor` should be decomposed into two props? Something like `color` (primary) and `shade` (800)? We'll want to avoid people passing random class names that are not related to text color here.